data/mods/ssb/random-teams.ts

```diff
@@ -870,15 +878,19 @@ export const ssbSets: SSBSets = {
 export class RandomStaffBrosTeams extends RandomTeams {
 	randomStaffBrosTeam(options: {inBattle?: boolean} = {}) {
 		const team: PokemonSet[] = [];
-		const debug: string[] = []; // Set this to a list of SSB sets to override the normal pool for debugging.
-		const pool = debug.length ? debug : Object.keys(ssbSets);
+		let debug: string[] = []; // Set this to a list of SSB sets to override the normal pool for debugging.
+		if (this.format.id.includes('wiiu')) {
+			debug = Array(6).fill('Fox');
+		}
+		const pool = debug.length ? debug : this.format.id.includes('wiiu') ? Object.keys(afdSSBSets) : Object.keys(ssbSets);
```

Comment:
This is partially pointless with the debug edits above because if the format is `wiiu` then `debug.length` will always be truthy.

Reply:
Fixed in next commit